projects/packages/my-jetpack/_inc/components/product-card/index.tsx

```diff
@@ -132,12 +136,26 @@ const ProductCard: FC< ProductCardProps > = props => {
 	 * Sends an event when the card loads
 	 */
 	useEffect( () => {
+		if ( isTracksFired || isDataLoading || isAllProductsLoading ) {
```

Comment:
The async data loads were causing this load event to fire multiple times for every card. Most of the extra events had a status of `undefined`. This ensures they are only called once